### ISSUE TITLE
Research: Automated tests for a11y

### ISSUE THREAD

**Issue by kulmann** [opencloud-eu/web#400]
### Initial Checklist

- [x] I understand this is a user story and questions should be posted in the [Community Discussions](https://github.com/orgs/opencloud-eu/discussions/)
- [x] I searched [issues](https://github.com/opencloud-eu/web/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### User Story

As OpenCloud, we want to be sure that our web ui doesn't violate accessibility so that we have certainty about our good accessibility.

### User Value

- Punish a11y violations
- ensure that there are no a11y regressions

### Acceptance Criteria

- find a suitable solution for automated accessibility testing
- barrier for writing accessibility tests should be low
- first step here to test would be opening page of file app / personal space
- testing situation would be contrasttest or focus management, testing elements like buttons, font etc.

Pointer: https://playwright.dev/docs/accessibility-testing maybe?
